src/release/actions.d

```diff
@@ -0,0 +1,169 @@
+/*******************************************************************************
+
+    Action System Definitions
+
+    Before any actual modification is done, all actions are collected and
+    presented to the user.
+
+    To enable this, every function returns (and optionall receives) an instance
+    of ActionList which the caller then appends to their instance and so on
+    so that a list of actions is accumulated.
+
+    That list will then be shown to the user and after a confirmation executed
+    in order of how they were appended.
```

Comment:
"in the order in which they were appended" -- if you want to be clearer and grammatically correct.